src/components/Editor/Tabs.js

```diff
@@ -76,10 +84,12 @@ class Tabs extends PureComponent<Props, State> {
   componentDidMount() {
     window.requestIdleCallback(this.updateHiddenTabs);
     window.addEventListener("resize", this.onResize);
+    getEditorContainer(window).addEventListener("resize", this.onResize);
   }
 
   componentWillUnmount() {
     window.removeEventListener("resize", this.onResize);
+    getEditorContainer(window).removeEventListener("resize", this.onResize);
```

Comment:
nit: i think its fine to do this directly in here
```suggestion
    document.querySelector(".editor-pane").removeEventListener("resizeonend", this.onResize);
```